lib/apiScenario/postmanCollectionRunnerClient.ts

```diff
@@ -933,10 +989,33 @@ try {
     if (prevMethod !== "delete") {
       scriptTypes.push("StatusCodeAssertion");
     }
-    const postScripts = this.generatePostScripts(scriptTypes, undefined, armTemplate);
-    if (postScripts.length > 0) {
-      PostmanHelper.addEvent(item.events, "test", postScripts);
+    let responseAssertion: any;
+    if (finalStateVia === "azure-async-operation") {
+      scriptTypes.push("ResponseDataAssertion");
+      responseAssertion = {
+        "200": [{ test: "/body/properties", expression: "to.be.not.undefined" }],
+      };
```

Comment:
Could you define an errorCode for each test? and also update here: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/api-scenario/references/ErrorCodeReference.md

Reply:
Seems this errorCode is for schema validator or example validator, did not find a place to define error code for Api test assertion

Reply:
Send https://github.com/Azure/azure-rest-api-specs/pull/21959 to update the errorCode docs